src/main/java/io/strimzi/kafka/bridge/http/HttpLogging.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2019, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+package io.strimzi.kafka.bridge.http;
+
+import io.vertx.core.http.HttpServerRequest;
+import io.vertx.core.http.HttpServerResponse;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Utility class for logging HTTP requests/responses
+ */
+public class HttpLogging {
+
+    private static final Logger log = LoggerFactory.getLogger(HttpLogging.class);
+
+    public static void logRequest(HttpServerRequest request) {
+        log.info("Request: method = {}, path = {}", request.method(), request.path());
+        log.debug("Request: headers = {}", request.headers());
+    }
+
+    public static void logResponse(HttpServerResponse response) {
+        log.info("Response: statusCode = {}, message = {}", response.getStatusCode(), response.getStatusMessage());
+    }
```

Comment:
Does these methods really need to be in a separate class?  It looks like the could be moved to `HttpUtils` to me.

Reply:
Good point. I didn't like adding a new class as well. I will move the methods and deleting this class.